lib/screens/v2/candidate_generator/widgets/departures.ex

```diff
@@ -176,6 +186,11 @@ defmodule Screens.V2.CandidateGenerator.Widgets.Departures do
     end
   end
 
+  def fetch_section_departures(_, _, _, _) do
+    # Only get to this pattern match if wayfinding_only_text is not nil, in which case, we fetch no departures
```

Comment:
suggestion: Rather than explaining this in a comment we could "explain" it via explicitly pattern-matching on `wayfinding_only: true`. In fact I think it might make more sense to reverse the order of these clauses so the "exceptional" behavior (wayfinding-only) comes first.

```elixir
def fetch_section_departures(%Section{wayfinding_only: true}, _, _, _), do: {:ok, []}

def fetch_section_departures(
  %Section{
    query: %Query{opts: opts, params: params},
    filters: filters,
    bidirectional: is_bidirectional
  },
  disabled_route_types,
  departure_fetch_fn,
  now
) do
  # ...
end
```

Reply:
good call, that is cleaner!